src/ansys/health/heart/parts.py

```diff
@@ -142,35 +149,132 @@ def __str__(self) -> str:
 
     def _to_dict(self) -> dict:
         """Get part information to reconstruct from a mesh file."""
-        info = {
+        data = {
             self.name: {
                 "part-id": self.pid,
                 "part-type": self._part_type.value,
+                "fiber": self.fiber,
+                "active": self.active,
+                "_version": __version__,
                 "surfaces": {},
             }
         }
 
-        info2 = {}
-        info2["surfaces"] = {}
+        data2 = {}
+        data2["surfaces"] = {}
 
         for surface in self.surfaces:
             if isinstance(surface, SurfaceMesh):
                 if surface.id:
-                    info2["surfaces"][surface.name] = surface.id
+                    data2["surfaces"][surface.name] = surface.id
 
         if hasattr(self, "caps"):
-            info2["caps"] = {}
+            data2["caps"] = {}
             for cap in self.caps:
-                info2["caps"][cap.name] = cap._mesh.id
+                data2["caps"][cap.name] = cap._mesh.id
 
         if hasattr(self, "cavity"):
-            info2["cavity"] = {}
+            data2["cavity"] = {}
             if self.cavity is not None:
-                info2["cavity"][self.cavity.surface.name] = self.cavity.surface.id
+                data2["cavity"][self.cavity.surface.name] = self.cavity.surface.id
+
+        data[self.name].update(data2)
 
-        info[self.name].update(info2)
+        return data
 
-        return info
+    def _get_predefined_surfaces(self) -> list[str]:
+        """Get a list of allowed surfaces for the part."""
+        return [
+            value.name for key, value in self.__dict__.items() if isinstance(value, SurfaceMesh)
+        ]
+
+    @staticmethod
+    def _set_from_dict(
+        data: dict, mesh: Mesh | None = None
+    ) -> Part | Septum | Chamber | Ventricle | Atrium | Artery | Myocardium:
+        """Reconstruct a part from a dictionary.
+
+        Parameters
+        ----------
+        data : dict
+            Dictionary that describes the part in JSON format.
+        mesh : Mesh, optional
+            Mesh object to use for reconstructing surfaces and cavities. If not provided,
+            the part will be created without surfaces and cavities.
```

Comment:
```suggestion
            the part is created without surfaces and cavities.
```
Avoid future tense if possible.